.github/workflows/tester.yml

```diff
@@ -0,0 +1,40 @@
+name: Run tests
```

Comment:
I wonder why do we need this one? It seems in the `main.yml` we already have all steps right?

Reply:
This one is particular for all `pull requests`. Therefore contributors know if there is an issue before actually merged.

Reply:
I can probably make this better I think by using
```
if: github.event_name == 'pull_request'
```

Reply:
Implemented in 20b22d66f6b1684e3ceec58e0db43be14e7d9339. This could be even better if Github supports to check another job's status before running a job. Currently, an extra ~25 seconds are used to build two Docker images which aren't actually used.